### DISCUSSION TITLE
After the clearning procedure of old firmware with T5UID1.CFG file, the touch screen is not work

### DISCUSSION THREAD

**Discussion by ACKET2009** [Desuuuu/DGUS-reloaded-Klipper#41] [Q&A]
Hello!
1) The first problem is, that after the upgrading of DWIN firmware on my CR-10S pro with clearning/formatting procedure by the special T5UID1.CFG file, the screen is loading, but the touch screen stopped to work... It seems, because the 13,14,22*.bin files is not present in archive.
2) The stop button on the screen stops the heaters, but do not stop the job
3) the bar of job status is empty while the job in process

**Comment by ACKET2009** [comment]
P.S. after formatting procedure there were 3 errors during the copy procedure - libraries, bmp and something else...
The answer was - where to get the files, that are not present in archive? and can I fix the problems of missfunction of communiction with klipper

**Comment by Desuuuu** [comment]
The installation instructions are available in the wiki [here](https://github.com/Desuuuu/DGUS-reloaded-Klipper/wiki/Flashing-the-firmware).

**Reply by ACKET2009** [reply]
Yes, I already find this *.bin files in that archive. But, before, I has download from github DGUS-reloaded-Klipper-master and try to flash it with [T5UID1.CFG] file. the memory was formatted and touchscreen becomes missfunctional... Only then I find the file by Your link and it was not a simple :) The screen is not work properly... I do not know - may be I do something incorrectly...
1) the digits near the clock icon during the print pocess shows something like that [0.0045678954 ] -I do not understand what it means...
2) the status bar of print job is emty during the job and only becomes full 100% when the job is done
3) the stop job button did not stop a whole job - it only stop the heaters
4) I fogot... will remeber and write soon... it was something else...

**Reply by ACKET2009** [reply]
May be some macros needed especially for the correct work of display?
